docs/guides/genomics-platform/analyzing-data/sequencerr.md

```diff
@@ -80,11 +69,60 @@ You can navigate to the **SequencErr** application [here](https://platform.stjud
     Set this to prevent large count files from being generated
     Default: true
 
-**Provide Input Parameters and Run the Analysis**
-![](../../../files/guides/tools/sequencerr/SequencErr_steps_7a-10.png)
+## Running the Analysis
+
+**SequencErr** application can be accessed from [https://platform.stjude.cloud/workflows/sequencerr](https://platform.stjude.cloud/workflows/sequencerr)
+
+!!! note
+    This tool is intended free-of-charge for non-profit usages. 
+    Please contact [Dr. Xiaotu Ma](mailto:Xiaotu.Ma@stjude.org) for for-profit usages and modifications
+
+Please refer to the stepwise **instructions PDF [here](../../../files/guides/tools/sequencerr/SequencErr_Instructions.pdf)** or following steps to learn how to launch the workflow, hook up input files, adjust parameters, run analysis and inspect output files.
+
+**Log in and Launch the (SequencErr)[https://platform.stjude.cloud/workflows/sequencerr]**
```

Comment:
This documentation is not just for reviewers; it's public for everyone. I recommend against having more than a note or two in there that directly addresses the reviewers. When someone needs to log in, you should refer them to our documentation on how to log in, not repeat the steps in your docs as well.

Reply:
As the St Jude account login option is not working, currently we have tailored login information for reviewers. Once this is working smoothly, it will be updated with general login instructions